tide_core.install

```diff
@@ -378,3 +378,41 @@ function tide_core_update_10012() {
   $shield_config = \Drupal::configFactory()->getEditable('shield.settings');
   $shield_config->set('paths', $paths_string)->save();
 }
+
+/**
+ * Adds link_list paragraph type to landing_page_component field.
+ */
+function tide_core_update_10013() {
+  $tideUpdateHelper = \Drupal::service('tide_core.entity_update_helper');
+  $configs = [
+    'paragraph.field_paragraph_link_list' => 'field_storage_config',
+    'link_list' => 'paragraphs_type',
+    'paragraph.link_list.field_paragraph_link_list' => 'field_config',
+    'paragraph.link_list.field_paragraph_title' => 'field_config',
+    'paragraph.link_list.default' => 'entity_form_display',
+  ];
+  foreach ($configs as $name => $type) {
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    if ($storage->load($name) !== NULL) {
+      continue;
+    }{
```

Comment:
These braces look a bit weird, probably not required.

Reply:
The magic of the php compiler, even though I wrote it wrong, it still compiled just fine... I actually meant to write an if-else.

Reply:
🤣